src/index.js

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
 import store from './store';
-import App from './components/App';
-import '../src/Main';
+import App from './container/App';
+import './Main.scss';
```

Comment:
We have resolved for .scss files to be loaded automatically. It is therefore okay to import `Main` instead of `Main.scss`.

Reply:
Hound is complaining when I remove the `scss` extension. Probably you could advice me how to go about this.